.github/workflows/ghcr.yml

```diff
@@ -35,11 +35,25 @@ jobs:
           DOCKER_BUILD_ORG=$(echo "${{ github.repository }}" | tr '[A-Z]' '[a-z]' | cut -d '/' -f 1)
           # Find directories containing Dockerfile but not containing .dockerfileignore
           while IFS= read -r dockerfile_dir; do
+          
             # Check if .dockerfileignore exists in the directory
-            if [ ! -f "$dockerfile_dir/.dockerfileignore" ]; then
-              # Change directory and execute 'make all'
-              pushd "$dockerfile_dir" > /dev/null
-              make all DOCKER_BUILD_ORG=$DOCKER_BUILD_ORG
-              popd > /dev/null
+            if [ -f "$dockerfile_dir/.dockerfileignore" ]; then
+                echo "$dockerfile_dir/.dockerfileignore exists, skipping build and push"
+                continue
             fi
+          
+            # Check if image was already exist in ghcr.io
+            pushd "$dockerfile_dir" > /dev/null
+            FULL_IMAGE=$(make get-full-image DOCKER_BUILD_ORG=$DOCKER_BUILD_ORG)
+            popd > /dev/null
+            EXISTS=$(docker manifest inspect "$FULL_IMAGE" > /dev/null 2>&1 && echo "true" || echo "false")
```

Comment:
Is this checking that the images have the same SHA? Or will it return true if there's an existing image with the same tag?

Reply:
Tag.

Reply:
I think that's an issue, no? Won't it keep us from overwriting the `latest` tag?

Or if we publish v.1.2.3, we will probably want to overwrite tags `v1` and `v1.2`

Reply:
you are right! 😭 this is a problem. we can overwrite `latest`, `v*`, `v*.*` each build

Reply:
<img width="803" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/16201837/b3887b77-e19f-4741-80d4-b513d0ceb8f9">
<img width="940" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/16201837/a01d9774-77c4-47c9-9f19-d63d985ab0ce">

I've just double-checked the workflow, it's ok no problem. it only gets the build version tag to compare, if match, it will trigger the build and push `version` tag and `latest` tag to ghcr.io

according to your advice, i added the `major` and `minor` tags when build. (but i commented out the major version right now, i think the v0 is not needed to push, don't you think?)

Reply:
<img width="752" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/16201837/ab4070b1-59ee-4103-947e-c859a4caaa15">